### PR TITLE
Adding custom upgrade jobs for PowerVS

### PR DESCRIPTION
**Pull request by varad-ahirwadkar**

### CODE REVIEW

File: jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-next-upgrade/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+            image 'quay.io/powercloud/inbound-agent:4.6-1.3'
+            args '-v /etc/resolv.conf:/etc/resolv.conf'
+            label 'jump-vpc-x86_64'
+        }
+    }
+    
+    parameters {
+        string(defaultValue: '', description: 'Current Build(quay image or build number)', name: 'CurrentBuild')
+        string(defaultValue: '', description: 'Upgrade Build(quay image or build number)', name: 'UpgradeBuild')
+        string(defaultValue: '', description: 'Current Build Release', name: 'Release')
```
Comment (Shilpi-Das1) [inline jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-next-upgrade/Jenkinsfile:15]: @varad-ahirwadkar Please add validation for release and upgrade release

File: jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-two-next-upgrade/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+            image 'quay.io/powercloud/inbound-agent:4.6-1.3'
+            args '-v /etc/resolv.conf:/etc/resolv.conf'
+            label 'jump-vpc-x86_64'
+        }
+    }
+    parameters {
+        string(defaultValue: '', description: 'Current Build(quay image or build number)', name: 'CurrentBuild')
+        string(defaultValue: '', description: 'Upgrade Build(quay image or build number)', name: 'UpgradeBuild')
+        string(defaultValue: '', description: 'Next Upgrade Build(quay image or build number)', name: 'NextUpgradeBuild')
+        string(defaultValue: '', description: 'Current Build Release', name: 'Release')
```
Comment (Shilpi-Das1) [inline jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-two-next-upgrade/Jenkinsfile:15]: Please add validation

**Review by Shilpi-Das1 — CHANGES_REQUESTED** [review]

File: jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-next-upgrade/Jenkinsfile (hunk 1 above)
Comment (varad-ahirwadkar) [reply jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-next-upgrade/Jenkinsfile:15]: ok sure

File: jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-two-next-upgrade/Jenkinsfile (hunk 2 above)
Comment (varad-ahirwadkar) [reply jobs/pipelines/powervs/custom-powervs-jobs/custom-ocp4x-powervs-sydney04-p9-min-two-next-upgrade/Jenkinsfile:15]: ok sure I will add

**Review by Shilpi-Das1 — APPROVED** [review]
/lgtm

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

File: jobs/pipelines/powervs/poll-powervs-images/Jenkinsfile [hunk 3]
```diff
@@ -18,8 +20,9 @@ pipeline {
     }
     environment {
         //users and credentials. All must be defined in Jenkins Credentials
-        IBMCLOUD_API_KEY = credentials('IBMCLOUD_UPSTREAM_CI_API_KEY')
-        PVSADM_VERSION = "v0.1.5"
+        IBM_CLOUD_API_KEY = credentials('IBMCLOUD_API_KEY')
```
Comment (Shilpi-Das1) [inline jobs/pipelines/powervs/poll-powervs-images/Jenkinsfile:23]: Please change IBMCLOUD_API_KEY to IBMCLOUD_UPSTREAM_CI_API_KEY

**Review by Shilpi-Das1 — CHANGES_REQUESTED** [review]

File: jobs/pipelines/powervs/poll-powervs-images/Jenkinsfile (hunk 3 above)
Comment (varad-ahirwadkar) [reply jobs/pipelines/powervs/poll-powervs-images/Jenkinsfile]: I have changed it

**Review by Shilpi-Das1 — APPROVED** [review]
/lgtm

**Pull request merged**